docs/src/components/ReanimatedAnimations/ReanimatedAnimationsSection/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import CodeBlock from '@theme/CodeBlock';
+import styles from './styles.module.css';
+import InteractiveExampleComponent from '../../InteractiveExample/InteractiveExampleComponent';
+import ArrowRight from '@site/static/img/examples/arrow-right-light.svg';
+
+interface Props {
+  title: string;
+  body: string;
+  docsLink: string;
+  component: React.ReactNode;
+  mobileComponent: React.ReactNode;
+  tabletComponent: React.ReactNode;
+  code: string;
+  idx: number;
+}
+
+export default function ReanimatedAnimationsSection({
+  title,
+  body,
+  docsLink,
+  component,
+  mobileComponent,
+  tabletComponent,
+  code,
+  idx,
+}: Props) {
+  const [isMobileView, setIsMobileView] = useState(window.innerWidth <= 420);
```

Comment:
```suggestion
  const [isMobileView, setIsMobileView] = useState(window.innerWidth <= MOBILE_SIZE);
```
extract magic numbers to something like `MOBILE_SIZE`, `TABLET_SIZE` so it's consistent throughout the file